redCOMPONENT-COM/redSHOP
# REDSHOP-5902 -  Fix securities alert at gitHub [Update to Gulp 4]

How to test:
1. `npm install -g n`
2. `sudo n 13.2`
3. `npm install --save-dev`
4. `npm rebuild node-sass`

Check version:
`node -v` => 13.2 is ok
![image](https://user-images.githubusercontent.com/6948022/76399125-b1680680-63b0-11ea-9a65-14b3681a95a5.png)

`gulp -v` => 4.0.2 is ok
![image](https://user-images.githubusercontent.com/6948022/76399203-d2c8f280-63b0-11ea-8076-5291534ac638.png)

Then check all Gulp command is run or not
